CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [0.6.0]
```

Comment:
Can you change this to `Unreleased` for now? I have some stuff I want to add before then next release